frontends/web/src/components/guide/guide.tsx

```diff
@@ -54,11 +59,35 @@ const Guide = ({ children, title = t('guide.title') }: TProps) => {
         <div className={style.content}>
           {children}
           <div className={style.appendix}>
-            {t('guide.appendix.text')}
-            {' '}
-            <A className={style.link} href="https://bitbox.swiss/support/">
-              {t('guide.appendix.link')}
-            </A>
+            {service ? (
+              <>
+                <span className={style.question}>{t('guide.appendix.questionService', { serviceName: service.name })}</span>
+                <br />
+                <br />
+                {t('guide.appendix.textService', { serviceName: service.name })}
+                {' '}
+                {service.linkDisplay.includes('@') ? (
+                  <span className={style.link}>
+                    {t('guide.appendix.linkService', { link: service.linkDisplay })}
+                  </span>
```

Comment:
I just realized that this is not ideal for RTL langauges. It was already wrong before, but with this PR it adds more trnaslations, it would be good to check if we can make it work with RTL before translators translate it.